pandas/tseries/tools.py

```diff
@@ -238,6 +238,18 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
         datetime strings, and if it can be inferred, switch to a faster
         method of parsing them. In some cases this can increase the parsing
         speed by ~5-10x.
+    origin : scalar convertible to Timestamp / string ('julian', 'epoch'),
+        default 'epoch'.
+        Define relative offset for the returned dates.
```

Comment:
I think that "Define relative offset for the returned dates" is not fully correct. As the offset you specify with `origin` is used to correctly parse the *input*, not the *returned dates*

So I would maybe replace it with something like

```
Date to parse numeric values as number of units (determined by `unit`) since this origin.
```

Does this sounds understandable to you? (english is also not my native tongue)